nifi-nar-bundles/nifi-metrics-reporting-bundle/nifi-metrics-reporting-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -14,12 +14,17 @@ The following binary components are provided under the Apache Software License v
     The following NOTICE information applies:
       Metrics
       Copyright 2010-2013 Coda Hale and Yammer, Inc., 2014-2017 Dropwizard Team
-
       This product includes software developed by Coda Hale and Yammer, Inc.
 
       This product includes code derived from the JSR-166 project (ThreadLocalRandom, Striped64,
       LongAdder), which was released with the following comments:
 
           Written by Doug Lea with assistance from members of JCP JSR-166
           Expert Group and released to the public domain, as explained at
-      http://creativecommons.org/publicdomain/zero/1.0/
\ No newline at end of file
+          http://creativecommons.org/publicdomain/zero/1.0/
+
+      The derived work in the nifi-metrics module is adapted from
```

Comment:
Is this the right place, or should it be in the nifi-metrics module?

Reply:
The `NOTICE` for `nifi-metrics-reporting-nar` needs to include an amended NOTICE since Dropwizard Metrics is included with the NAR:
```
[INFO] -------------< org.apache.nifi:nifi-metrics-reporting-nar >-------------
[INFO] Building nifi-metrics-reporting-nar 1.10.0-SNAPSHOT                [5/5]
[INFO] --------------------------------[ nar ]---------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:2.10:tree (default-cli) @ nifi-metrics-reporting-nar ---
[INFO] org.apache.nifi:nifi-metrics-reporting-nar:nar:1.10.0-SNAPSHOT
[INFO] +- org.apache.nifi:nifi-metrics-reporting-task:jar:1.10.0-SNAPSHOT:compile
[INFO] |  +- io.dropwizard.metrics:metrics-graphite:jar:4.1.0:compile
[INFO] |  |  \- io.dropwizard.metrics:metrics-core:jar:4.1.0:compile
[INFO] |  \- org.apache.nifi:nifi-metrics:jar:1.10.0-SNAPSHOT:compile
[INFO] |     \- io.dropwizard.metrics:metrics-jvm:jar:4.1.0:compile
[INFO] \- org.apache.nifi:nifi-metrics-reporter-service-api-nar:nar:1.10.0-SNAPSHOT:compile
[INFO] ------------------------------------------------------------------------
```